src/measure_theory/measure/measure_space.lean

```diff
@@ -3424,6 +3424,21 @@ piecewise_ae_eq_of_ae_eq_set hst
 lemma indicator_meas_zero (hs : μ s = 0) : indicator s f =ᵐ[μ] 0 :=
 (indicator_empty' f) ▸ indicator_ae_eq_of_ae_eq_set (ae_eq_empty.2 hs)
 
+lemma indicator_ae_eq_of_ae_eq {g : α → β} (hfg : f =ᵐ[μ] g) :
+  s.indicator f =ᵐ[μ] s.indicator g :=
```

Comment:
This should hold for all filters, not only the ae filter. Should probably be named `eventually_eq.indicator`, and put in `order.filter.indicator_function`. No need to give a specialized version for the ae filter, as the general version will apply directly. Same thing for the next lemma.